host/bcm283x/gpio_test.go

```diff
@@ -156,10 +158,15 @@ func TestPinPWM(t *testing.T) {
 
 	clockMemory = &clockMap{}
 	pwmMemory = &pwmMap{}
+	dmaBufAllocator = func(s int) (*videocore.Mem, error) {
```

Comment:
You can put that into init() for the test.

Reply:
done.